packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -269,7 +271,11 @@ export class HttpServer {
     this.setupResponseLogging();
     this.setupGracefulShutdownHandlers();
 
-    const staticAssets = new StaticAssets(basePathService, config.cdn);
+    const staticAssets = new StaticAssets(
+      basePathService,
+      config.cdn,
+      this.env.packageInfo.buildShaShort
+    );
```

Comment:
NIT: is 3 or 4 the team's consensus on the limit of named parameters before we switch to kwargs 😄?

```ts
    const staticAssets = new StaticAssets({
      basePath: basePathService,
      cdnConfig: config.cdn,
      buildSha: this.env.packageInfo.buildShaShort
    });
```

But it's totally optional (and probably just a personal preference)

Reply:
Kwarg time!